qcodes/tests/dataset/test_dataset_loading.py

```diff
@@ -29,6 +29,14 @@ def test_load_by_id():
     assert loaded_ds.completed is False
     assert loaded_ds.exp_id == 1
 
+    # let's take a reasonably arbitrary run number that is quite definitely not
```

Comment:
I think this comment is confusing. Perhaps simply writing "let's take a run number that is not" is better?

Reply:
sure! and i changed the number to `1` as well.